tests/copy-paste.js

```diff
@@ -685,13 +685,13 @@ define([
             paste([
                 ["id", "type", "labelItext:en-default", "itemsetData"],
                 ["/select", "SelectDynamic", "select",
-                 '[{"instance":null,"nodeset":"/items","labelRef":"@name","valueRef":"@id"}]'],
+                 '[{"instance":null,"nodeset":"/items","labelRef":"name","valueRef":"@id"}]'],
```

Comment:
I reverted this change and dug into this some more, and I'm a bit confused about what this test should be doing. Stepping through this test, it looks like:
- `paste` calls `deserialize`
- In `deserialize`, because the given DynamicSelect mug doesn't have any children, `afterDynamicSelectInsert` is called
- `afterDynamicSelectInsert` creates a new itemset mug. The new mug doesn't have a label or value (is this the problem? should the new itemset be populated based off of the `{"instance":null,"nodeset":"/items","labelRef":"name","valueRef":"@id"}` supplied when the test calls `paste`?), so its value and label get set based on the first data source (happily, `getDataSources` does have the async data available when it's called).

So the @name/@id labelRef and valueRef are indeed being ignored, but I'm not sure which part of the code should be taking them into account. Thoughts?

I'm also a little confused because it doesn't look like vellum supports the exact use case of this test (copying a DynamicSelect without children). In the UI, adding a dynamic select automatically adds an itemset child that can't be deleted, so you can't copy/paste only the select.